app/src/main/java/nl/tudelft/b_b_w/model/block/BlockFactory.java

```diff
@@ -45,7 +45,7 @@ public static final Block createBlock(BlockData data) throws HashException {
     @Deprecated
     public static final Block getBlock(String type, String newOwner, int sequenceIndex, String hash,
```

Comment:
Yeah I understand that it will be merged into develop. But I didn't know you would refactor it in another PR. Clear now. Will approve